packages/ag-charts-enterprise/src/axes/angle/angleAxis.ts

```diff
@@ -60,10 +60,7 @@ export abstract class AngleAxis<
     protected tickData: AngleAxisTickDatum<TDomain>[] = [];
     protected radiusLine: _Scene.Path = this.axisGroup.appendChild(new Path());
 
-    constructor(moduleCtx: _ModuleSupport.ModuleContext, scale: TScale) {
-        super(moduleCtx, scale);
-        this.includeInvisibleDomains = true;
-    }
+    override includeInvisibleDomains = true;
```

Comment:
Could we avoid  introducing `override` properties, the semantics aren't exactly clear - I'd prefer the constructor approach we had before (maybe make it an option to the super-class constructor for cleaner configuration)?